themes/dark/lxqt-panel.qss

```diff
@@ -458,6 +457,123 @@ QMenu::indicator:non-exclusive:checked {
     color: white;
 }
 
+/*
+ * Fancy menu
+ */
+#FancyMenu {
+    margin: 2px;
+    padding: 2px;
+    border: none;
+    background: rgba(0, 0, 0, 80%);
+    qproperty-icon: url(mainmenu.svg);
+    color: white;
+}
+
+LXQtFancyMenuWindow {
+    background: rgba(0, 0, 0);/*disabled transparency because buggy*/
+}
+
+#FancyMenu QLineEdit {
+    background: black;
+    padding: 3px;
+    margin-bottom: 1px;
+    color: white;
+    selection-background-color: #640b0c;
+}
```

Comment:
By not adding border, you've left it to the widget style. It may be OK with Fusion, but it isn't OK with some Kvantum themes (which are light and have completely rounded borders). Please change it as

```css
#FancyMenu QLineEdit {
    background: black;
    border: 1px solid #3c3c3c;
    padding: 3px;
    margin-bottom: 1px;
    color: white;
    selection-background-color: #640b0c;
}
```

I've just added `border: 1px solid #3c3c3c;` from `#MainMenu QLineEdit`.